bundles/org.openhab.binding.electroluxappliances/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,472 @@
+			<channel id="applianceUiSwVersion" typeId="applianceUiSwVersion"/>
+			<channel id="applianceTotalWorkingTime" typeId="applianceTotalWorkingTime"/>
+			<channel id="optisenseResult" typeId="optisenseResult"/>
+			<channel id="detergentExtradosage" typeId="detergentExtradosage"/>
+			<channel id="softenerExtradosage" typeId="softenerExtradosage"/>
+			<channel id="waterUsage" typeId="waterUsage"/>
+			<channel id="totalWashCyclesCount" typeId="totalWashCyclesCount"/>
+			<channel id="connectionState" typeId="connectionState"/>
+		</channels>
+
+		<properties>
+			<property name="vendor">Electrolux</property>
+			<property name="thingTypeVersion">1</property>
+		</properties>
+
+		<representation-property>serialNumber</representation-property>
+
+		<config-description>
+			<parameter name="serialNumber" type="text" required="true">
+				<label>Serial Number</label>
+				<description>The appliance serial number.</description>
+			</parameter>
+		</config-description>
+
+	</thing-type>
+
+
+	<channel-type id="status">
+		<item-type>String</item-type>
+		<label>Current Status</label>
+		<description>Information on current status.</description>
+		<state readOnly="true" pattern="%s"/>
+	</channel-type>
+
+	<channel-type id="temperature">
+		<item-type>Number:Temperature</item-type>
+		<label>Temperature</label>
+		<description>The measured temperature.</description>
+		<category>Temperature</category>
+		<state readOnly="true" pattern="%.1f %unit%">
+		</state>
+	</channel-type>
+
+	<channel-type id="humidity">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Humidity</label>
+		<description>The measured humidity.</description>
+		<category>Humidity</category>
+		<state readOnly="true" min="0" max="100" pattern="%d %unit%"/>
+	</channel-type>
+
+	<channel-type id="tvoc">
+		<item-type>Number:Dimensionless</item-type>
+		<label>TVOC</label>
+		<description>The total Volatile Organic Compounds.</description>
+		<state readOnly="true" pattern="%d %unit%"/>
+	</channel-type>
+
+
+	<channel-type id="pm1">
+		<item-type>Number:Density</item-type>
+		<label>PM1</label>
+		<description>The Particulate Matter 1 (0.001mm).</description>
+		<state readOnly="true" pattern="%d %unit%"/>
+	</channel-type>
+
+	<channel-type id="pm2_5">
+		<item-type>Number:Density</item-type>
+		<label>PM2.5</label>
+		<description>The Particulate Matter 2.5 (0.0025mm).</description>
+		<state readOnly="true" pattern="%d %unit%"/>
+	</channel-type>
+
+	<channel-type id="pm10">
+		<item-type>Number:Density</item-type>
+		<label>PM10</label>
+		<description>The Particulate Matter 10 (0.01mm).</description>
+		<state readOnly="true" pattern="%d %unit%"/>
+	</channel-type>
+
+	<channel-type id="co2">
+		<item-type>Number:Dimensionless</item-type>
+		<label>CO2</label>
+		<description>The measured CarbonDioxide.</description>
+		<state readOnly="true" pattern="%d %unit%"/>
+	</channel-type>
+
+	<channel-type id="filterLife">
+		<item-type>Number:Dimensionless</item-type>
```

Comment:
I guess percentage and not hours or other?
```suggestion
		<item-type unitHint="%">Number:Dimensionless</item-type>
```

Reply:
Correct! 
Fixed!